actions/channel_actions.ts

```diff
@@ -162,14 +168,14 @@ export function addUsersToChannel(channelId, userIds) {
     };
 }
 
-export function unmuteChannel(userId, channelId) {
+export function unmuteChannel(userId: UserProfile['id'], channelId: Channel['id']) {
     return ChannelActions.updateChannelNotifyProps(userId, channelId, {
         mark_unread: NotificationLevels.ALL,
-    });
+    } as ChannelNotifyProps);
 }
 
-export function muteChannel(userId, channelId) {
+export function muteChannel(userId: UserProfile['id'], channelId: Channel['id']) {
     return ChannelActions.updateChannelNotifyProps(userId, channelId, {
         mark_unread: NotificationLevels.MENTION,
-    });
+    } as ChannelNotifyProps);
```

Comment:
I guess this is needed because we need to pass a partial ChannelNotifyProps and we are missing values. If that is the case, shouldn't `updateChannelNotifyProps` be asking for a Partial one, instead of the whole thing?

Reply:
this is a really good suggestion i dont why i missed this one